### DISCUSSION TITLE
Mantendo dinheiro recebido por serviço em dólares e a contabilidade por trás

### DISCUSSION THREAD

**Discussion by igorlvicente** [DyegoCosta/trabalhando-remoto#122] [Q&A]
Algum de vocês têm experiência em fazer serviço para fora e manter parte disso em uma conta em dólares (Payoneer ou TransferWise, por exemplo)?

Como é a contabilidade nesses casos?

**Comment by samiraguiar** [comment]
https://github.com/DyegoCosta/trabalhando-remoto/issues/102#issuecomment-711023834

**Reply by igorlvicente** [reply]
Quando você converte parte do valor em euros para reais, você tem uma taxa de conversão. Você usa essa mesma taxa para declarar o valor que você não converteu?

E depois, quando você usa o valor em Euros, como você calcula a "nova taxa"?

**Reply by samiraguiar** [reply]
Não, eu emito a NF com o valor cheio assim que recebo o pagamento em euro ou quando gero o invoice. Por exemplo, se recebi 100 euros pelo serviço, emito a NF de 100 euros, independente de quando (e quanto) eu for transferir para o Brasil

**Reply by igorlvicente** [reply]
Pelo que eu entendi, você faz como nesse exemplo:
1. Recebe 100 euros no Payoneer
2. Converte 90 e deixa 10
    > Digamos que na hora da conversão/invoice, você conseguiu uma taxa de 5 reais
3. Emite nota de 500 reais

Daí, digamos que depois de uns dias você resolve gastar seus 10 euros. Nesse dia, a taxa está 6 reais. O que você faz?
E se, ao invés de gastar, você quer converter para reais. O que você faria nesse caso também?

**Reply by samiraguiar** [reply]
> Digamos que na hora da conversão/invoice, você conseguiu uma taxa de 5 reais
> 3. Emite nota de 500 reais

Na verdade eu não uso a taxa de conversão da Payoneer pra definir o valor da cotação, uso a cotação PTAX de compra do euro do último dia útil anterior à emissão da nota

> Daí, digamos que depois de uns dias você resolve gastar seus 10 euros. Nesse dia, a taxa está 6 reais. O que você faz?

Eu gasto normalmente, porque o imposto em cima desses 10 euros já foi pago quando eu declarei a NF dos 100 euros lá no início.

> E se, ao invés de gastar, você quer converter para reais. O que você faria nesse caso também?

Converto normalmente, porque já foi declarado antes

**Reply by heliomeiralins** [reply]
> Eu gasto normalmente, porque o imposto em cima desses 10 euros já foi pago quando eu declarei a NF dos 100 euros lá no início.

O imposto já está pago, ok. Mas e a contabilidade da empresa? Esse dinheiro vai sair como distribuição de lucros? Como vc registra essa saída?

**Reply by samiraguiar** [reply]
@heliomeiralins se for mandar o dinheiro pra uma conta PF no exterior, eu imagino que teria que ser feita a distribuição sim. Mas não faço ideia de como seria feito nesse caso. Já pesquisei e não consegui encontrar muita coisa, mas sei que a isenção de impostos na distribuição de lucros não é aplicável a empresas offshore, então imagino que deva haver algumas condições quando ocorre a transferência de valores em outra moeda da PJ pra PF.

**Reply by danisteffen** [reply]
@samiraguiar , você conseguiu descobrir nesses 2 anos como fazer para mandar o dinheiro para uma conta PF no exterior? Ok, lanço como distribuição de lucros, mas como registro essa saída se não há registro numa conta bancária associada?

**Comment by RangelReale** [comment]
Parece que a opção mais certa legalmente é, por exemplo, usando o banco BS2:

* Pagamento em dólares -> Banco BS2 PJ Internacional (em dólar)
* Banco BS2 PJ Internacional (em dólar)  -> Banco BS2 PF Internacional (em dólar)

a partir do Banco BS2 PF Internacional, voce pode fazer o que quiser, mandar parte em dolar para um lugar, parte em real para outro, a partir daí é livre.

Tem umas taxas ai no meio, mas parece que para ser 100% legal precisa desses passos.

**Reply by kuwakino** [reply]
(mar/2022) taxa de transferência do BS2 para outros bancos: US$12
BS2 para BS2 é de graça.

**Comment by RangelReale** [comment]
Parece que tem outro problema com isso, pelo que entendi do [site da contabilizei](https://www.contabilizei.com.br/contabilidade-online/quanto-minha-empresa-de-servicos-paga-de-impostos-no-exterior/), só há isenção de PIS, Cofins e ISS se o dinheiro for enviado para o Brasil:

```
prestação de serviços para pessoa física ou jurídica residente ou domiciliada no exterior, cujo pagamento represente ingresso de divisas
```

abri um chamado com eles para confirmar.

**Reply by samiraguiar** [reply]
@RangelReale depois avisa aqui o que eles responderem, por favor.

Mas tem uma [instrução normativa](http://normas.receita.fazenda.gov.br/sijut2consulta/link.action?idAto=90994) que diz o seguinte:

> Art. 1º Os recursos em moeda estrangeira relativos aos recebimentos de exportações brasileiras de mercadorias e de serviços para o exterior, realizadas por pessoas físicas ou jurídicas, poderão ser mantidos em instituição financeira no exterior, observados os limites fixados pelo Conselho Monetário Nacional (CMN).
> ...
> Art. 3º Sobre as receitas mantidas no exterior na forma prevista no art. 1º, decorrentes da prestação de serviços para pessoa física ou jurídica residente ou domiciliada no exterior, não incidem a Contribuição para o PIS/Pasep e a Contribuição para o Financiamento da Seguridade Social (Cofins).
> § 1º Para fins de aplicação da alíquota zero da Contribuição para o PIS/PASEP e da Cofins incidentes sobre as receitas financeiras decorrentes de variações monetárias, em função da taxa de câmbio, de operações de exportação de bens e serviços para o exterior a que se refere inciso I do § 3º do art. 1º do Decreto nº 8.426, de 1º de abril de 2015, devem ser consideradas as variações cambiais ocorridas até a data da liquidação do contrato de exportação ou a data do recebimento pelo exportador dos recursos decorrentes da exportação.
> § 2º O benefício de que trata o § 1º não alcança as variações cambiais ocorridas após a data de recebimento, pelo exportador, dos recursos decorrentes da exportação.

Só que aí sobre essa questão de tributar a variação monteária, teve uma [mudança](https://lopescastelo.adv.br/receita-esclarece-pis-cofins-sobre-exportacoes/) em que a RFB disse que a receita financeira decorrente da variação cambial sofreria incidência do PIS e do Cofins:
> A Receita Federal cobrará 4,65% de PIS e Cofins das empresas que recebem pagamento por exportações em moeda estrangeira, mantêm o dinheiro no exterior e registram variação cambial positiva sobre esse montante. O entendimento foi publicado ontem por meio do Ato Declaratório Interpretativo (ADI) nº8.

Mas parece que o STF [já decidiu](https://costamonteiroadvocacia.com/2021/04/16/receita-de-variacao-cambial-de-exportacao-e-imune-a-pis-e-cofins-diz-stf/) que essa receita financeira não pode ser tributada:
> O Supremo Tribunal Federal (STF), em julgamento de recurso extraordinário com repercussão geral reconhecida pelo Plenário Virtual, entendeu que as receitas de exportação decorrentes da variação cambial não devem ser tributadas pelo Programa de Integração Social (PIS) e Contribuição para o Financiamento da Seguridade Social (Cofins). A decisão, tomada por unanimidade, negou provimento ao Recurso Extraordinário (RE) 627815, no qual a União questionava acórdão do Tribunal Regional Federal da 4ª Região (TRF-4) favorável a uma indústria paranaense do ramo ceramista.

**Comment by RangelReale** [comment]
Resposta da Contabilizei:

"O ingresso de divisas significa o cliente do exterior realizar o pagamento em moeda estrangeira em nome da sua empresa, após o recebimento o fato de converter em reais ou manter no exterior não interfere."

**Reply by kuwakino** [reply]
Complementando com uma outra resposta da Contabilizei:

> Para manter o dinheiro no exterior, a conta precisa estar no nome da empresa. Nesse caso, você precisa nos enviar o extrato mensalmente para contabilizarmos.
> 
> Se a conta no exterior estiver em nome da Pessoa Física, não aconselhamos que você mantenha o valor nela, pois refere-se valores recebidos pela prestação de serviços de sua empresa.
> 
> Todo valor recebido, aqui ou no exterior, precisa ser na conta da sua empresa e quanto a tributação, permanecerá sendo realizada normalmente sobre o valor total das notas fiscais emitidas, ou seja, mesmo que o valor não seja convertido para uma conta nacional, não desobriga a emissão da Invoice e nota fiscal da prestação do serviço.

**Comment by kuwakino** [comment]
Vi outra possibilidade de recebimento: Cobre.PJ da Contabilizei.

Eles criam um link de cobrança para pagamento em cartão de crédito - e aceitam pagamentos de cartões internacionais (em dólar).
(estou verificando taxas - atualizo nas respostas dessa)

**Reply by kuwakino** [reply]
O valor que cai é na conta em reais:

> Para você não terá diferenças no valor a receber, as taxas serão as mesmas que você tem para cobranças nacionais, mas o impacto será para seu cliente. 
>  
> O seu cliente irá receber o link para pagamento todo em português e no valor da cobrança que você emitiu, ele pagará em reais e no máximo em 1x. 
>  
> Qual será o impacto para seu cliente?
>  
> O cartão do cliente fará o câmbio de forma automática para real e irá cobrar as taxas extras.

**Comment by herculan0** [comment]
to no bs2 por causa da integraçao com a contabilizei, alguem sabe um fluxo
melhor? tava pensando em receber pelo husky que a taxa é menor porém nao
sei se vale tanto a pena devido a essa integraçao da contabilizei, por
causa dos impostos etc

Em seg., 28 de mar. de 2022 às 11:56, Eduardo Kuwakino <
***@***.***> escreveu:

**Reply by kuwakino** [reply]
Cada mês precisa ver qual compensa mais.

Tem respostas aqui (github) e no slack, vou deixar 2 threads do slack:
https://trabalho-remoto.slack.com/archives/C6WCM0SUQ/p1644606587586799
https://trabalho-remoto.slack.com/archives/C6W799KB3/p1646673268915869

**Comment by danisteffen** [comment]
Estou em dúvida se posso usar o Payoneer da minha empresa e não converter para reais todo o saldo, enviar para o Brasil e fazer o envio da retirada de lucros para mim numa data posterior.
Tem IOF para converter e enviar do Payoneer para o banco da empresa (uso o Cora), depois paga IOF para enviar pelo Remessa Online para a minha conta nos EUA.
Teria como fazer a retirada do Payoneer da minha empresa para a minha conta pessoal nos EUA, lançando como retirada de lucros na contabilidade da minha empresa, sem precisar fazer essa rota que demora mais tempo e paga 2x IOF?
Minha contabilidade está atualmente com a Contabilivre. Eles não sabem me responder essa pergunta...
